pylint/reporters/ureports/nodes.py

```diff
@@ -16,6 +16,9 @@
 """
 
 
+from typing import Optional
```

Comment:
```suggestion
from typing import Optional
```
The empty lines are not really needed.